app/mailers/emailer.rb

```diff
@@ -31,13 +38,14 @@ def new_feature_adoption_to_admin
 
   def user_email_verification(user)
```

Comment:
Can't this email and this method be removed entirely?

Reply:
Thats being handled in SRCH-1076. I am focusing on the requirements for this ticket.